src/main/java/net/minecraftforge/client/model/MultiModel.java

```diff
@@ -227,20 +227,30 @@ public ItemOverrideList getOverrides()
     private final ResourceLocation location;
     @Nullable
     private final IModel base;
-    private final IModelState baseState;
     private final Map<String, Pair<IModel, IModelState>> parts;
 
+    // for binary compatibility
```

Comment:
If these aren't intended to be kept, except temporarily for compatibility, can you deprecate them and add a `TODO: remove` comment?

Reply:
Whole class is deprecated - for at least 3 major MC versions. But I can add TODO.

Reply:
If the whole thing is deprecated, then it's not such an issue.